packages/runtime/test/transport/files.test.ts

```diff
@@ -338,23 +338,23 @@ describe("Load peer file with token reference", () => {
         const response = await transportServices2.files.getOrLoadFile({ reference: token.truncatedReference });
 
         expect(response).toBeSuccessful();
-        expect(response.value).toMatchObject({ ...file, isOwn: false });
+        expect(response.value).toMatchObject({ ...file, isOwn: false, ownershipToken: undefined });
```

Comment:
Hm, we should do a quick call about all these added lines in the existing tests.